src/components/ReportWelcomeText.js

```diff
@@ -54,18 +65,26 @@ function ReportWelcomeText(props) {
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const isChatRoom = ReportUtils.isChatRoom(props.report);
     const isDefault = !(isChatRoom || isPolicyExpenseChat);
+    const isAdminsOnlyPostingRoom = ReportUtils.isAdminsOnlyPostingRoom(props.report);
+    const isAnnounceRoom = ReportUtils.isAnnounceRoom(props.report);
     const participantAccountIDs = lodashGet(props.report, 'participantAccountIDs', []);
     const isMultipleParticipant = participantAccountIDs.length > 1;
     const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(
         OptionsListUtils.getPersonalDetailsForAccountIDs(participantAccountIDs, props.personalDetails),
         isMultipleParticipant,
     );
-    const roomWelcomeMessage = ReportUtils.getRoomWelcomeMessage(props.report);
+    const isUserPolicyAdmin = PolicyUtils.isPolicyAdmin(props.policy);
+    const roomWelcomeMessage = ReportUtils.getRoomWelcomeMessage(props.report, isUserPolicyAdmin);
     const moneyRequestOptions = ReportUtils.getMoneyRequestOptions(props.report, participantAccountIDs, props.betas);
+
     return (
         <>
             <View>
-                <Text style={[styles.textHero]}>{props.translate('reportActionsView.sayHello')}</Text>
+                <Text style={[styles.textHero]}>
```

Comment:
```suggestion
                <Text style={[styles.textHero]}>{props.translate('reportActionsView.welcomeToRoom', {roomName: ReportUtils.getReportName(props.report)})}</Text>
```

Reply:
This will impact on all chat reports.

<img width="1822" alt="Screenshot 2023-08-22 at 1 10 56 AM" src="https://github.com/Expensify/App/assets/104348397/6f28b212-46f0-4fad-9d7c-e72e6dfbc68a">

Reply:
Are you sure we want to update this? @eVoloshchak

Reply:
ofcourse, we can use that but I think only #announce and room with admins only posting are in scope of the PR. Thats why I use the above logic. I'll make changes. Please confirm if thats the only change I need to make?

Reply:
>  I think only #announce and room with admins only posting are in scope of the PR

I'm using [this](https://github.com/Expensify/App/issues/23819#issuecomment-1661146413) comment as a reference, but yeah, it's not 100% clear

- > Update Say Hello to Welcome to #announce!
- > For everyone: Only admins can send messages in this channel.

I think we should show `Welcome to #roomName!` for all rooms, not just for announce for consistency sake
@joelbettner, could you chime in on this? Thanks

Reply:
> I think we should show Welcome to #roomName! for all rooms

Yes, I think that sounds good, @eVoloshchak @ishpaul777.